test/test_bert_tokenizer.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 import transformers
 from onnxruntime_extensions import PyOrtFunction, BertTokenizer, util
+from transformers import BertTokenizerFast
```

Comment:
Does BertTokenizer support offset_mapping?

Reply:
not the regular one - only the HuggingFace Fast tokenizers support offset_mapping.